libs/dh/market-participant/actors/feature-actors/src/lib/drawer/dh-actor-drawer.component.html

```diff
@@ -0,0 +1,72 @@
+<!--
+@license
+Copyright 2020 Energinet DataHub A/S
+
+Licensed under the Apache License, Version 2.0 (the "License2");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<ng-container *transloco="let t; read: 'marketParticipant.actorsOverview.drawer'">
+  <watt-drawer #drawer (closed)="onClose()">
+    <watt-drawer-topbar>
+      <dh-actor-status-badge [status]="actor?.status" />
+    </watt-drawer-topbar>
+
+    <watt-drawer-heading>
+      <h2 class="actor-heading">{{ actor?.name }}</h2>
```

Comment:
Suggestion for another day. Maybe we should have a title / label on the watt-drawer-heading component, and move the h tag into the component. 
I never know what heading is right to put in, this would fix that :-)

Reply:
Good idea. Yeah, so far we've used `h1`, `h2` and `h3`. So I think we're all confused as to which heading level to use :D